modules/samples-operator-overview.adoc

```diff
@@ -47,3 +47,18 @@ the following upon deletion:
 
 Upon deletion of the samples resource, the Samples Operator recreates the
 resource using the default configuration.
+
+[id="samples-operator-bootstrapped"]
+== Samples Operator bootstrapped as removed
+
+Certain circumstances will result in the Samples Operator bootstrapping itself as removed including:
+
+* If the Samples Operator cannot reach link:registry.redhat.io[registry.redhat.io] after three minutes on initial start-up after installation.
+* If the Samples Operator detects it is on an IPv6 network.
+* If global proxy is configured.
+* If the Samples Operator detects it is installed on IBM Z.
+
+[IMPORTANT]
+====
+IPv6 installations are not currently supported by link:registry.redhat.io[registry.redhat.io].
```

Comment:
and that matters because the bulk of the sample imagestream images are by default pulled from there ... I'll defer to you, but that additional detail might be helpful